bitnami/owncloud/templates/deployment.yaml

```diff
@@ -1,143 +1,176 @@
-              name: {{ include "owncloud.databaseSecretName" . }}
-              key: mariadb-password
-{{- $port:=.Values.service.port | toString }}
-        - name: OWNCLOUD_HOST
-          value: "{{ include "owncloud.host" . }}{{- if ne $port "80" }}:{{ .Values.service.port }}{{ end }}"
-        - name: OWNCLOUD_USERNAME
-          value: {{ default "" .Values.owncloudUsername | quote }}
-        - name: OWNCLOUD_PASSWORD
-          valueFrom:
-            secretKeyRef:
-              name: {{ template "owncloud.fullname" . }}
-              key: owncloud-password
-        - name: OWNCLOUD_EMAIL
-          value: {{ default "" .Values.owncloudEmail | quote }}
-        {{- if .Values.extraEnvVars }}
-        {{- include "owncloud.tplValues.render" ( dict "value" .Values.extraEnvVars "context" $ ) | nindent 8 }}
+        {{- include "certificates.initContainer" . | nindent 8 }}
+        {{- if .Values.initContainers }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
         {{- end }}
-        ports:
-        - name: http
-          containerPort: 80
-        livenessProbe:
-          httpGet:
-            path: /status.php
-            port: http
-            httpHeaders:
-            - name: Host
-              value: {{ include "owncloud.host" . | quote }}
-          initialDelaySeconds: 120
-          timeoutSeconds: 5
-          failureThreshold: 6
-        readinessProbe:
-          httpGet:
-            path: /status.php
-            port: http
-            httpHeaders:
-            - name: Host
-              value: {{ include "owncloud.host" . | quote }}
-          initialDelaySeconds: 30
-          timeoutSeconds: 3
-          periodSeconds: 5
-        resources:
-{{ toYaml .Values.resources | indent 10 }}
-        volumeMounts:
-        {{- include "certificates.volumeMount" . | indent 8 }}
-        - name: owncloud-data
-          mountPath: /bitnami/owncloud
-{{- if .Values.metrics.enabled }}
-      - name: metrics
-        image: {{ template "owncloud.metrics.image" . }}
-        imagePullPolicy: {{ .Values.metrics.image.pullPolicy | quote }}
-        command: [ '/bin/apache_exporter', '--scrape_uri', 'http://status.localhost:80/server-status/?auto']
-        ports:
+      containers:
+        - name: owncloud
+          image: {{ template "owncloud.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.args "context" $) | nindent 12 }}
+          {{- end }}
+          env:
```

Comment:
I'm missing BITNAMI_DEBUG here when image.debug is set, could you add it? It's pretty useful sometimes.